src/app/games/hive/tests/HiveDummyMinimax.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { Player } from 'src/app/jscaip/Player';
+import { HiveDummyMinimax } from '../HiveDummyMinimax';
+import { HiveMove } from '../HiveMove';
+import { HivePiece } from '../HivePiece';
+import { HiveNode, HiveRules } from '../HiveRules';
+import { HiveState } from '../HiveState';
+
+describe('HiveDummyMinimax', () => {
+
+    let rules: HiveRules;
+    let minimax: HiveDummyMinimax;
+
+    beforeEach(() => {
+        rules = HiveRules.get();
+        minimax = new HiveDummyMinimax(rules, 'HiveDummyMinimax');
+    });
+    it('should propose 5 moves at first turn', () => {
+        // Given the initial state
+        const state: HiveState = HiveState.getInitialState();
+        const node: HiveNode = new HiveNode(state);
+
+        // When computing the list of moves
+        const moves: HiveMove[] = minimax.getListMoves(node);
+
+        // Thne there should be 5 moves
+        expect(moves.length).toBe(5);
```

Comment:
Eeeeh thneuuuuuu ! :D
I guess aussi qu'on pourrais pinailler et dire "should be 5 drop, one by different kind of piece" mais EH :D

Reply:
sneeeeeeee